src/test/resources/debug-config/logback-service.xml

```diff
@@ -5,15 +5,25 @@
             <pattern>[%date{ISO8601}] %-5level %msg%n</pattern>
         </encoder>
     </appender>
-    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+    <appender name="FILE-DOWNLOAD" class="ch.qos.logback.core.FileAppender">
```

Comment:
Same comment here as above, I don't think renaming this is necessary. I'd prefer keeping the appender names consistent across modules.

Reply:
done